briefcase-vs-aggregate.rst

```diff
@@ -0,0 +1,51 @@
+**********************************
+Briefcase vs Aggregate
+**********************************
```

Comment:
Please change this to the underline style:

```
Title
=======
```

The over-under style is for larger docs that *might* be broken up in the future. 
When the over-under style is used, the first subsection headline with the `===` underline.